docs/spec/index.md

```diff
@@ -1,54 +1,67 @@
 
-| **Field Name** | **Type** | **Required** | **Description** |
-| --- | --- | --- | --- |
-| deadhead_id | ID | Required | Identifies a deadhead. |
-| service_id | ID referencing [**calendar.service_id**](https://developers.google.com/transit/gtfs/reference#calendartxt) | Required | Identifies a set of dates when the deadhead is scheduled to take place. |
-| block_id | ID | Required | Identifies the block to which the deadhead belongs. |
-| shape_id | ID referencing [**shapes.shape_id**](https://developers.google.com/transit/gtfs/reference#shapestxt) | Optional | Identifies a geospatial shape that describes the vehicle travel path for a deadhead. |
-| to_trip_id | ID referencing [**trips.trip_id**](https://developers.google.com/transit/gtfs/reference#tripstxt) | Conditionally Required | Identifies the trip scheduled immediately following to the deadhead within the block_id. |
-| from_trip_id | ID referencing [**trips.trip_id**](https://developers.google.com/transit/gtfs/reference#tripstxt) | Conditionally Required | Identifies the trip scheduled immediately prior to the deadhead within the block_id. |
-| to_deadhead_id | ID referencing **deadheads.deadhead_id** | Conditionally Required | Identifies the deadhead scheduled immediately following the deadhead within the block_id. |
-| from_deadhead_id | ID referencing **deadheads.deadhead_id** | Conditionally Required | Identifies the deadhead scheduled immediately prior to the deadhead within the block_id. |
 
-### ops_locations.txt
+### Files
 
-| **Field Name** | **Type** | **Required** | **Description** |
-| --- | --- | --- | --- |
-| ops_location_id | ID | Required | Identifies an operational location. |
-| ops_location_code | String | Optional | Short text or a number that identifies the operational location for internal use. |
-| ops_location_name | String | Required | Name of the operational location. |
-| ops_location_desc | String | Optional | Description of the operational location. |
-| ops_location_lat | Latitude | Required | The latitude of the operational location. |
-| ops_location_lon | Longitude | Required | The longitude of the operational location. |
+| **File Name** | **Type** | **Description** |
+| --- | --- | --- |
+| trips_supplement.txt | Supplement | Supplements and modifies GTFS [trips.txt](https://github.com/google/transit/blob/master/gtfs/spec/en/reference.md#tripstxt) with deadheads and other non-public trip information.|
+| stops_supplement.txt | Supplement | Supplements and modifies GTFS [stops.txt](https://github.com/google/transit/blob/master/gtfs/spec/en/reference.md#stopstxt) with internal stop locations, waypoints, and other non-public stop information.|
+| stop_times_supplement.txt | Supplement | Supplements and modifies GTFS [stop_times.txt](https://github.com/google/transit/blob/master/gtfs/spec/en/reference.md#stop_timestxt) with non-public times that trips stop at locations, and related information. |
```

Comment:
How about "with non-public times **_at which_** trips stop at locations, and related information" (emphasis added)?

The point is that it doesn't include the locations, but rather the non-public times that they're stopping at these locations.

Reply:
I think Josh's point was that this file also includes new stop_times for new non-public locations, not just modifications to stop_times at public locations. The docs could suggest that use case. (I see how you're trying to avoid making it sound like this file contains the data about those locations, cuz that's in stops_supplement, but I don't think that'll be confusing.)